e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -543,26 +539,26 @@ func (s *IbcEurekaTestSuite) TestICS20Transfer() {
 		err = json.Unmarshal(returnPacket.Data, &transferPacketData)
 		s.Require().NoError(err)
 		s.Require().Equal(denomOnCosmos.GetFullDenomPath(), transferPacketData.Denom)
-		s.Require().Equal(sdkTransferAmount.String(), transferPacketData.Amount)
-		s.Require().Equal(s.UserB.FormattedAddress(), transferPacketData.Sender)
-		s.Require().Equal(strings.ToLower(userAddress.Hex()), transferPacketData.Receiver)
+		s.Require().Equal(transferAmount.String(), transferPacketData.Amount)
```

Comment:
Just putting this in the middle here, but I love all the cleanups here. The renaming and variables is a big improvement! <3